### ISSUE TITLE
TWE: Create Template Issue: [TYPE & TIMING] Interviews: Make Issues

### ISSUE THREAD

**Issue by bradyse** [hackforla/internship#321]
### Overview
We need to create a template issue for an epic issue that tracks all the interview issues so that we have consistent, transparent, defined and self-supporting structure for people creating them.

### Action Items
- [ ] Add to the Project Board
- [ ] Add the milestone: Project Management
- [ ] Add a feature label specific to what the template is related to, if available.  If not clear what you should use, ask product.
- [ ] Any place you see `[replace...]` replace both the text and the brackets with what it is asking for
- [ ] Create a draft of new template by copying the content block below to a comment on this issue, which will be known as our Draft
- [ ] In your draft, Fill out the top part (called front matter) with the info it needs.  See the Resources section below for definition and example.
- [ ] Define any Dependencies (or remove the dependency section)
- [ ] Customize the Overview section
- [ ] Fill out the Action Items section (in your draft) by defining the steps that are required to be done, keeping the following in mind:
   - [ ] The person working on this issue might be new to the org, so the issue should be able to be worked on without any prior knowledge
   - [ ] If the person needs to add links, or put documents in specific locations, the place where they can do that action must be linked in the resources section
      - [ ] the links provided by the issue creator should be in the same order as the Action Items 
- [ ] If documents or folders (sheets, docs, presentations, dashboard, etc.) will be created, find the document template that should be used by checking the WIKI Templates by role pages.  
  - [ ] if there is a template
     - [ ] Add a link to the template into the ` ### Resources`  section of your draft
  - [ ] if there is no template yet
    - [ ] put a placeholder into the` ### Resources`  section of the draft `[TWE: Template: [replace with name of type]]`
- [ ] Get sign-off by product on the draft
- [ ] Create the template issue on the repo
- [ ] Update the appropriate place in wiki for your template (see resources section)

#### WIKI Templates
- by role pages
  - [TWE: WIKI: UX Research Guides Templates](https://github.com/hackforla/internship/wiki/UX-Research-Guides-Templates) 
  - [Design]
  - [Product]
  - [Engineering]
- default document templates, in case you don't have one yet can found on the [Design Process Overview page](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

#### Content to copy for draft
```
Draft of [replace with name of type] template issue
---
name: TWE: Create Template Issue: [replace with name of template]
about: 
title: TWE: [replace with action verb] [replace with name of template]
labels: 'feature: board / google drive / tools', 'feature: missing', 'size: missing', 'role: missing', 'milestone: missing'
assignees: 
---
# Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- [ ] Find WIKI page for this issue by looking at role pages [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team) - then add the appropriate link to the resources section
- [ ] Find document template that should be used [Templates & Product Branding](https://github.com/hackforla/internship/wiki/Design-Process-Overview) - then add that specific template link into the resources section
- [ ] If no dependencies, delete dependency section
- [ ] any place you see `[replace...]` replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] add appropriate labels and remove the corresponding missing labels
- [ ] delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
WE NEED TO DO X FOR Y REASON

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, etc.

### Resources
```
### Resources
<details>
<summary>Front Matter Fields and Meaning</summary>

- name: (name that appear on the new issue tab)
- about: (is the description that appears next to the issue template on the new issue tab
- title: (the title that appears when the issue is created.  User then customizes this title for their issue)
- labels: (choose the labels that will be the same each time, and choose missing labels for the ones that depend on content)
- assignees: 
</details>

<details>
<summary>Front Matter Field example</summary>

- name: 'Research: Roadmap'
- about: The start of a new roadmap
- title: 'TWE: Research Roadmap [name of type]'
- labels: 'feature: research, feature: roadmap, milestone: missing, role: UI/UX research,
-   size: 0.50pt'
- assignees: ''
</details>

**Comment by bradyse** [comment]
Draft of Research: Interviews: Make Issues template issue
---
name: TWE: Create Template Issue: [TYPE & TIMING] Interviews: Make Issues
about: The parent issue that tracks issues for all interviews of a specific research plan
title: TWE: Research: Interviews: Make Issues
labels: 'feature: board / google drive / tools', 'feature: research', 'feature: missing', 'size: 1pt', 'role: UX research lead', 'milestone: missing'
assignees: 
---
# Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- [ ] Find WIKI page for this issue by looking at role pages [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team) - then add the appropriate link to the resources section
- [ ] Find document template that should be used [Templates & Product Branding](https://github.com/hackforla/internship/wiki/Design-Process-Overview) - then add that specific template link into the resources section
- [ ] If no dependencies, delete dependency section
- [ ] any place you see `[replace...]` replace both the text and the brackets with what it is asking for (including in the front matter above but not where it says name, but not where it says title.
- [ ] add appropriate labels and remove the corresponding missing labels
- [ ] delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
WE NEED TO DO X FOR Y REASON

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, etc.

### Resources

**Comment by bradyse** [comment]
1. Progress: 
- Working on developing an interview training guide template based upon RP6's training guide that can be added to this template issues list of resources.
- Working on organizing template wiki information that can be referenced in this issue for further streamlining
3. Blockers: None
4. Availability: 1-3 hours/week
5. ETA: Next week
6. Pictures or links (if necessary):

_* remember to add links to the top of the issue if they are going to be needed again._

**Comment by bradyse** [comment]
@priyatalwar @sharongupta1 @ExperimentsInHonesty 

This issue template draft is complete and should be ready to be added to the templates on the repo. See [comment with draft](https://github.com/hackforla/internship/issues/321#issuecomment-1259814771).

**Comment by bradyse** [comment]
Sara and Bonnie will test this before it's signed off.

**Comment by pandanista** [comment]
Draft of Template Issue: Research: Interviews: Make Issues
---
name: TWE: Create Template Issue: RP[###] [TYPE OF PARTICIPANT] [TYPE OF INTERVIEW (E.G., INTAKE OR EXIT)] Interviews: Make Issues and Track Completion
about: The parent issue that tracks issues for all interviews of a specific research plan
title: TWE: Research: Interviews: Make Issues
labels: 'feature: board / google drive / tools', 'feature: research', 'feature: missing', 'size: 1pt', 'role: UX research lead', 'milestone: missing' 'epic' 
assignees: 
---

### Dependencies
- [ ] An interview training guide has been created and approved [replace with - ISSUE LINK AND ADD LINK TO GUIDE IN RESOURCES]

### Overview
We need to create an issue for each [replace with - TYPE OF PARTICIPANT] [replace with - TYPE OF INTERVIEW (E.G., INTAKE OR EXIT)] interview to make sure we are getting all the steps uniform.

### Action Items
- [ ] **Customize the issue settings**. On the right column of this issue page, change the issue settings to ensure the issue is properly categorized and easy to manage.
     - [ ] Under Assignees, assign yourself.
     - [ ] Under Labels, add label of the correct research plan number associated with this research plan (e.g., Research: RP001) (helps you find related issues in the same workflow) and remove the corresponding missing label
     - [ ] Under Projects, add to the Project Management Board.
- [ ] **Customize the issue content**. Any place you see `[replace...]` replace both the text and the brackets with what it is asking for
     - [ ] Customize the title
          - [ ] Add the appropriate research plan number (e.g., RP006)
          - [ ] Replace TYPE OF PARTICIPANT with the participant type (e.g., Intern)
          - [ ] Replace TYPE OF INTERVIEW with the type of interview (e.g., Intake or Exit)
     - [ ] Customize the resources
          - [ ] Go to the wiki page: Research Output Overview (# 1 link in resources) and find the relevant wiki page for this specific research plan. Then, change the [RP__ Wiki Page] in the Resources below with this RP's specific wiki page.
          - [ ] Find the Google Drive link folder for your research plan  (e.g., Research by Type>2. Intern>RP6 - Intern Intake Interviews)
               - [ ] Go to the Google Drive's Research by Type Folder (# 3 link in Resources)
               - [ ] Go into the relevant Type Folder (e.g., 1. Mentor, 2. Intern, etc.) 
               - [ ] Once inside, right-click on the relevant destination folder and copy the link. (e.g., RP6 - Intern Intake Interviews)
               - [ ] Paste the Google Drive link to resource # 4 and revise the display text to include the Research Plan number (e.g., `Google Drive RP__ Folder` to `Google Drive RP008 Folder`)
          - [ ] Inside this Research Plan's Google Drive folder, copy the link to the Interview Instructions document.
               - [ ] Paste the link to the resources of this issue (Resource # 5)
               - [ ] Paste the link to the template text (Resource # 11, Item A) that you will create the issues for the UX researchers.
          - [ ] In Resources, click on # 6, TWE: Interview Session and Roll Call Table Template
               - [ ] Click on the `Use Template` button to save it to your own drive
               - [ ] Move the file into the Internship's shared drive in this Research Plan's Google Drive folder (Resource # 4).
          - [ ] In Resources, click on 1password link (# 7) to find the login information within the Internship-UX vault for Easy Retro
               - [ ] Log into EasyRetro using the login information.
               - [ ] Create an Easy Retro for Q&A that occurs during the interview. 
               - [ ] Save that link to the resources of this issue (Resource # 8)
               - [ ] Copy link to the template issue in gray (Resource # 11, Item C) to be added to the UX researchers' issues for future reference.
          - [ ] Click on the Google Form Template (Resource # 9) and move this template into the internship-ux@hackforla.org personal drive
               - [ ] Customize Google Form with the appropriate information, including the Interview Session Table Spreadsheet.
               - [ ] Ensure that the recordings will be saved directly to the correct internship-ux personal drive folder
- [ ] **Create an issue for each interview.**
     - [ ] IS[YY] [TYPE & TIMING] Interviews: [Type] # 1
     - [ ] IS[YY] [TYPE & TIMING] Interviews: [Type] # 2
     - [ ] Replace this action item with as many interview issues that you need. EXPLAIN  HOW TO DO THIS BY CLICKING ON THE ACTION ITEM
     - [ ] Add the issue numbers/links to the interview spreadsheet (Resource # 6)
- [ ] **Schedule the interviews** (Note if you have to create more than one interview, you may have to do the following steps several times)
     - [ ] Go to the UI/UX wiki page for setting up interviews on Hack for LA Zoom accounts (Resource Link 10).
     - [ ] Copy the action items to this issue in a comment and check off each item.
     - [ ] Update the related interview issue with the time and Zoom link information.
- [ ] **Follow-up steps**
     - [ ] Check whether all recordings have been uploaded to the correct folder and that any personally identifiable information is NOT stored in the shared drive.
     - [ ] Contact the UX researchers who uploaded the recordings and ask them to delete their files. 

### Resources
1. [Wiki: Research Output Overview](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
1. [RP__ Wiki Page] Use the link above to find the correct research plan and once the link has been added here, add the link to the template above.
1. [Google Drive Research by Type Folder](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=sharing)
1. [Google Drive RP__ Folder]
1. [TWE: RP__ Interview Instructions](https://docs.google.com/document/d/1YKJZHejAAvX19NfPh1AKC4k3CuX0nrJQq9N3BXBAB2U/template/preview) - find in this research plan's Google Drive folder and replace this with the appropriate link
1. [TWE: Template: Interview Session Roll Call Spreadsheet](https://docs.google.com/spreadsheets/d/1dmJJLaBDBiZWy4-dmdxG3XqP4OemR5aJTOScD_GsoLc/template/preview) - (replace with the correct spreadsheet for this research plan here, then add to the link in the template above)
1. [1password.com](https://1password.com/)
1. [Easy Retro Link] - Use 1password for Easy Retro account instructions and replace this info with the Easy Retro link
1. [Interview Recording Upload Form Template](https://docs.google.com/forms/d/1FsDgBvpkjfJzmkxFl80I9HWenoOwpvst7oMAaDFgPEg/edit) - move to internship-ux personal Google Drive and customize template with appropriate links and folder information necessary for the file upload question
1. [UI/UX Wiki: Setup Interviews on Hack for LA Zoom Accounts](https://github.com/hackforla/UI-UX/wiki/Setup-Interviews-on-Hack-for-LA-Zoom-accounts)

#### Resource # 11 Template for IS[YY] [TYPE & TIMING] Interview Issues
```
### Overview
We need to track the interview for [Intern's Number] so that we don't drop the ball or lose data about it.

### Action Items
- [ ] If you are conducting the interview on your own and not in a breakout room
     - [ ] 5 min prior, login to 1password and get hostkey for Zoom [Zoom meeting room#]
     - [ ] Join the meeting via the calendar link in your invite or here in this issue
     - [ ] Claim host
- [ ] When they enter room, begin consent process, and ask if we can record and transcribe.
- [ ] Turn on transcript.
- [ ] Use script in interview.
- [ ] Upload recording and transcript and a chat file if applicable using the Interview Recording Upload Form

### Resources/Instructions
#### Interview Links and Instructions
- A) [RP__: Interviews Instructions]
- B) [TWE: RP__: [TYPE & TIMING] Interview Session and Roll Call Table]
- C) [Easy Retro link for Q&A that occurs during interview]
- D) [[TYPE & TIMING Interview Upload Form]

#### Interview details
- Zoom link for interview
- Date and time for interview:

#### Other links
- [Code for America's Guide on Qualitative Research](https://info.codeforamerica.org/qualitative-research)

#### Text once you are ready to send invite update
We have added the zoom invite info and the panelist for your interview.  Looking forward to seeing you! 
```

**Comment by bradyse** [comment]
### Meeting notes from 2022-11-14
- Move "Customize issue content" to first set of instructions
- Change general instructions to say "Customize RP Number, Type of Interview, and Type of Participant
- Double check that keywords are appearing correctly when searched for within the browser (make sure that we didn't miss any)
- Move Resource # 9 up the hierarchy in level of heading so that the bullet point says "Customize additional documents" because this specific resource involves instructions for modifying the document itself.
- Fix formatting so that the levels of heading are more clearly delineated and separate from the checkboxes.

**Comment by ExperimentsInHonesty** [comment]
This needs to be made into an issue

### Dependencies
ANY ISSUE NUMBERS THAT ARE BLOCKERS OR OTHER REASONS WHY THIS WOULD LIVE IN THE ICEBOX

### Overview
WE NEED TO DO X FOR Y REASON

### Action Items
- [ ] log into internship-ux@hackforla.org google drive
- [ ] create a folder for the uploads
- [ ] Click on the Google Form Template
- [ ] Copy this template into the internship-ux@hackforla.org personal drive in the new folder
- [ ] Customize the following information in the Google Form
- [ ] Modify name of Google Form (add RP[Replace 000]) to file name
- [ ] Modify TYPE OF PARTICIPANT with participant type (e.g., "Mentor") in the dropdown question
- [ ] Modify URL in the question marked `Get your UXR id from this spreadsheet` to link to correct Interview Session Roll Call Spreadsheet, which can be found in the RP[Replace 000] folder
   - [ ] Open the Google Drive's Research by Type Folder (# ? link in Resources)
   - [ ] Open the [Replace TYPE OF PARTICIPANT] Folder (e.g., 1. Mentor, 2. Intern, etc.) 
   - [ ] Open the [Replace 00] folder
- Edit Response Destination
  - [ ] In Form edit, go to Responses
  - [ ] click 3 dots
  - [ ] click Select Response Destination
  - [ ] choose the folder you created in the internship-ux@hackforla.org google drive
- [ ] Add the link to the form in the resources below

### Resources/Instructions
- [Interview Recording Upload Form Template](https://docs.google.com/forms/d/1FsDgBvpkjfJzmkxFl80I9HWenoOwpvst7oMAaDFgPEg/edit)

**Comment by ExperimentsInHonesty** [comment]
This needs to be made into an issue

Title: TWE: RP[Replace 000] Interview Session Roll Call Spreadsheet: Create

TWE: Interview Session and Roll Call Table Template

### Dependencies
ANY ISSUE NUMBERS THAT ARE BLOCKERS OR OTHER REASONS WHY THIS WOULD LIVE IN THE ICEBOX

### Overview
WE NEED TO DO X FOR Y REASON

### Action Items
- [ ] Click on TWE: Template: Interview Session Roll Call Spreadsheet (resources)
- [ ] Click on the Use Template button to save it to your own drive
- [ ] Move the file into the Internship's shared drive in this Research Plan's Google Drive folder

### Resources/Instructions
- [TWE: Template: Interview Session Roll Call Spreadsheet](https://docs.google.com/spreadsheets/d/1dmJJLaBDBiZWy4-dmdxG3XqP4OemR5aJTOScD_GsoLc/template/preview)

**Comment by ExperimentsInHonesty** [comment]
This needs to be made into an issue template

Create Easy Retro
Log into 1 password, Internship-UX vault 
get link to easy retro account for internship-ux
 Log into EasyRetro using the login information.
 Create an Easy Retro for Q&A that occurs during the interview.

**Comment by ExperimentsInHonesty** [comment]
Notes for testing
When trying to do this part 
Customize No. 2.03: TWE: RP006 Interview Instructions document link
I could not find the document directly inside the RP006 folder, I had to go one more folder deep to [Training Deck and Related Materials](https://drive.google.com/drive/folders/1_FyHqyiWstqKclyzTaJcF22P9OQcKJwe)

**Comment by ExperimentsInHonesty** [comment]
@bradyse this issue still has the ready for milestone label on it.  But I see there are items not checked off in the notes section.  Are those things for me to do?  If it is not, please remove the ready for milestone label.

**Comment by ExperimentsInHonesty** [comment]
The latest work to be done on this issue is in this comment https://github.com/hackforla/internship/issues/321#issuecomment-1314699229 as unresolved.  Pasting here to reduce the time spent searching around

#### Customize No 2.04: TWE: RP006: Interview Session Roll Call spreadsheet link
- We are calling the document: Interview Session Roll Call spreadsheet and the template is called 1.04. [TWE: Template: Interview Session Roll Call Spreadsheet](https://docs.google.com/spreadsheets/d/1dmJJLaBDBiZWy4-dmdxG3XqP4OemR5aJTOScD_GsoLc/template/preview)
- In RP006 folder the document is called: TWE: RP6: Intern Intake Interview Session Table
- Should we rename the document in all the folders and any links in their related wikis?
    - [ ] Finalize the direct object part of the file name. Currently we're going with the convention `TWE: Role: Direct Object: Action Verb`. Therefore, the template here would be `TWE: Template: [Direct object]` 
    - [ ] Update existing session table docs to reflect the new naming structure (search in Drive for "session")

#### Customize No 2.06: Easy Retro
- I created a really rough draft of what that issue template would contain.  See this comment https://github.com/hackforla/internship/issues/321#issuecomment-1314462552
- Lets discuss and determine if we want to make an issue for that?
   - [ ] Make another issue-template issue for creating an Easy Retro

**Comment by ExperimentsInHonesty** [comment]
Once you finish the renaming, add dependency label and move to the icebox

**Comment by pandanista** [comment]
@ExperimentsInHonesty: An update regarding the action items from this [comment](https://github.com/hackforla/internship/issues/321#issuecomment-1371605498):

- The renaming of the Roll Call table is done. See my [comment](https://github.com/hackforla/internship/issues/343#issuecomment-1376489047) in #343.
- I don't recall if we have a conclusion on whether to make an issue for No. 2.06 Easy Retro or not. Should we discuss it on 1/11, Wed, 10 AM (PT)?
- Should we add dependency label and move this issue to the icebox now or waiting on the template issue on Easy Retro?
